classes/sso/user/mumie_user_service.php

```diff
@@ -55,6 +55,17 @@ public static function get_user(string $moodleid, \stdClass $mumietask) : mumie_
         return new mumie_user($moodleid, $mumieid);
     }
 
+    /**
+     * Get the Problem selector user from a Moodle user id.
+     *
+     * @param string $moodleid The Moodle user id
+     * @return mumie_user The Problem selector user
+     */
+    public static function get_problem_selector_user(string $moodleid) : mumie_user {
```

Comment:
The parameter name moodleid is a bit confusing for me. Imo this parameter has nothing to do with a user, if it is named this way.

Reply:
The name `moodleid` is used for the Moodle user id in `sso_service.php`, `mumie_user.php` and `mumie_user_service.php`. So I also use this name here for the sake of consistency. I don't know enough about the project, in order to make an informed decision whether this name is appropriate or should be refactored in all of its occurrences. So my suggestion would be, to keep it as is.